### ISSUE TITLE
Option menu not displayed on page reload

### ISSUE THREAD

**Issue by Nzel96** [victoryhb/streamlit-option-menu#68]
I am running into an issue when I reload the page the option_menu disappears.
I am not getting any errors of any message. 
Can anyone help with this issue?

Here is my environment info
streamlit==1.33.0   
streamlit-option-menu==0.3.12

**Comment by KeanuReeves007** [comment]
Yes, I've experienced this issue as well. My authentication is implemented through streamlit_authenticator. I've observed that when I reload the page, the logout in the session_state is None. At that point, I call st.run().

eg:
`
    if st.session_state["authentication_status"] and st.session_state["logout"] is None:
        st.session_state["logout"] = True
        st.rerun()
`

This should solve the problem, although I'm not sure what caused it. I hope this helps you.
